extensions/json-language-features/client/src/jsonClient.ts

```diff
@@ -163,6 +164,13 @@ export async function startClient(context: ExtensionContext, newLanguageClient:
 		window.showInformationMessage(l10n.t('JSON schema cache cleared.'));
 	}));
 
+	toDispose.push(workspace.onDidSaveTextDocument((textDocument) => {
+		const sortOnSave = workspace.getConfiguration().get<boolean>(SettingIds.enableSortOnSave);
+		if (sortOnSave && (textDocument.languageId === 'json' || textDocument.languageId === 'jsonc')) {
+
```

Comment:
Something seems to be missing here?

Reply:
Hey sorry, it's a draft, and still work in progress